src/main/scala/KotlinPlugin.scala

```diff
@@ -68,6 +68,7 @@ object KotlinPlugin extends AutoPlugin {
           classDirectory.value, streams.value)
     }.dependsOn (compileInputs in (Compile,compile)).value,
     compile := (compile dependsOn kotlinCompile).value,
-    kotlinSource := sourceDirectory.value / "kotlin"
+    kotlinSource := sourceDirectory.value / "kotlin",
+    definedTests in Test := KotlinTest.kotlinTests.value
```

Comment:
will this clobber existing definitions of definedTests? if so, perhaps the correct approach would be to use `++=` rather than `:=`